### ISSUE TITLE
Accessibility: View title bar is not accessible with Orca

### ISSUE THREAD

**Issue by rebornix** [microsoft/vscode#91602]
Re #91063

- OS: Ubuntu 18.04
- Screen Reader: Orca

Bug:

* Open File Explorer
* Tab through the Views, Open Editors, Explorer, Outline, Timeline
* Orca doesn't read the name for each view title, while Voice Over on macOS does.

**Comment by rebornix** [comment]
Didn't reproduce this issue in Debug View.

**Comment by isidorn** [comment]
Interesting. Assigning to march to investigate more then.

**Comment by isidorn** [comment]
@rebornix this is a good catch.
The issue here is that the `Outline` and `Timeline` view do not specify an `ariaHeaderLabel` when they are being constructed.

This issue was that attribute was not mandatory, thus a lot of clients simply did not pass it.
I think I have simplified things quite a bit by computing the `ariaLabel` from the `title` and removing the `ariaHeaderLabel` attribute to not be controlled by clients.
Now all views proprely have an `ariaLabel` set.

fyi @sandy081 @sbatten @joaomoreno
